pkg/cmd/container/list.go

```diff
@@ -77,18 +78,36 @@ func filterContainers(ctx context.Context, client *containerd.Client, filters []
 		}
 	}
 
+	var wg sync.WaitGroup
+	cMap := make(map[string]string)
+	var mu sync.Mutex
+	// formatter.ContainerStatus(ctx, c) is time consuming so we do it in goroutines and return the container's id with status as a map.
+	// prepareContainers func will use this map to avoid call formatter.ContainerStatus again.
+	for _, c := range containers {
+		wg.Add(1)
```

Comment:
```suggestion
                c := c
		wg.Add(1)
```

the above line code simplifies the code and avoid using a mutex, right ?

Reply:
@ningmingxiao WDYT ?

Reply:
> avoid using a mutex

I don't think it will avoid the mutex. Current write to a map still need the mutex.

Reply:
indeed map are note safe for [concurrent](https://go.dev/doc/faq#atomic_maps) write. thanks @djdongjin  for your pointer ;)